### PR TITLE
Add organization user invitation email template

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
### Proposed changes in this pull request
- $subject

image

### CODE REVIEW

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 1]
```diff
@@ -17195,4 +17195,469 @@
...
+                                                                        valign="top" align="left">
+                                                                        <p style="font-size: 14px; line-height: 140%; padding-top: 8px;">
+                                                                            <span
+                                                                                style="font-size: 12px; line-height: 16.8px; color: #808080;">You
+                                                                                received this email because you have an account in the
+                                                                                organization
+                                                                                <b>{{organization-name}}</b>. If you encounter any issues,
+                                                                                you may contact us at
+                                                                                <a
+                                                                                    class="wso2_orange3" href="mailto:{{organization.support.mail}}"
```
Comment (dewniMW) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:17633]: organization.support.mail -> what's the placeholder value for this?

Comment (ShanChathusanda93) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:17633]: This placeholder is an existing one. Seems like for the other places also this is not properly populated.

image

**Comment by AnuradhaSK** [comment]
Add a screenshot of the email into the description

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 1 above)
Comment (madurangasiriwardena) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:17633]: This placeholder is available to be configured with branding

**Review by madurangasiriwardena — DISMISSED** [review]

**Review by madurangasiriwardena — APPROVED** [review]

**Pull request merged**